src/service/translate.js

```diff
@@ -709,6 +710,29 @@ function $translate($STORAGE_KEY, $windowProvider, $translateSanitizationProvide
 
   /**
    * @ngdoc function
+   * @name pascalprecht.translate.$translateProvider#useForceAsyncReload
+   * @methodOf pascalprecht.translate.$translateProvider
+   *
+   * @description
+   * If force async reload is enabled, async loader will be always called
+   * even if $translationTable already contains the language key.
+   *
```

Comment:
This goes here: 
https://github.com/angular-translate/angular-translate/blob/master/src/service/translate.js#L236